install.sh

```diff
@@ -280,6 +294,64 @@ get_distribution() {
 	echo "$lsb_dist"
 }
 
+# Check if systemd is available and running
+has_systemd() {
+	command_exists systemctl && systemctl --version >/dev/null 2>&1
+}
+
+# Start and enable Docker daemon service
+start_docker_daemon() {
+	if is_dry_run; then
+		echo "# DRY RUN: Would start and enable Docker daemon service"
+		if has_systemd; then
+			echo "# DRY RUN: systemctl start docker"
+			echo "# DRY RUN: systemctl enable docker"
+		else
+			echo "# DRY RUN: service docker start"
+			echo "# DRY RUN: chkconfig docker on (or equivalent)"
+		fi
+		return
+	fi
```

Comment:
Instead of hardcoding the logic for try run, could we just override `sh_c` so it only prints the actual command?